### PR TITLE
W-15454595 Exchange non-managed instances jh

### PR DESCRIPTION
**Pull request by JennyHajee**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/to-configure-api-settings.adoc [hunk 1]
```diff
@@ -16,6 +16,19 @@ In a private Exchange instance, asset administrators have access to change the v
 
 REST APIs provide a mocking service with data to test API features. The mocking service is always public.
 
+== Configure Non-Managed Instances 
+For published assets not managed by API Manager, asset administrators can add, edit, and delete non-managed instances that are serving AsyncAPIs, GraphQL and REST APIs. The name, URL, and visibility of a non-managed instance can be changed.
```
Comment (mcampo) [inline modules/ROOT/pages/to-configure-api-settings.adoc]: Can we fix the wording here? API Manager does not manage assets, it manages instances.

Comment (mcampo) [reply modules/ROOT/pages/to-configure-api-settings.adoc]: maybe just a small addition:
```
For published assets with instances not managed by API Manager, ...
```
but I'm not sure if that reads well

Comment (JennyHajee) [reply modules/ROOT/pages/to-configure-api-settings.adoc]: Yes, this is good. Fixed

**Review by mcampo — DISMISSED** [review]
👍

**Review by mcampo — DISMISSED** [review]

File: modules/ROOT/pages/to-configure-api-settings.adoc [hunk 2]
```diff
@@ -16,6 +16,18 @@ In a private Exchange instance, asset administrators have access to change the v
 
 REST APIs provide a mocking service with data to test API features. The mocking service is always public.
 
+== Configure Non-Managed Instances 
+Exchange enables you to configure APIs with instances not managed by API Manager. Add, edit, and delete non-managed instances that are serving AsyncAPIs, GraphQL, REST, and HTTP APIs. You can change the name, URL, and visibility of a non-managed instances.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-configure-api-settings.adoc]:
```suggestion
Exchange enables you to configure APIs with instances not managed by API Manager. Add, edit, and delete non-managed instances that are serving AsyncAPIs, GraphQL, REST, and HTTP APIs. You can change the name, URL, and visibility of non-managed instances.
```

File: modules/ROOT/pages/to-configure-api-settings.adoc [hunk 3]
```diff
@@ -16,6 +16,18 @@ In a private Exchange instance, asset administrators have access to change the v
...
+
+. Select the asset.
+. In the navigation bar, select *API Instances*.
```
Comment (hannanelson) [inline modules/ROOT/pages/to-configure-api-settings.adoc:25]: Is this the left nav or something else?

File: modules/ROOT/pages/to-configure-api-settings.adoc [hunk 4]
```diff
@@ -16,6 +16,18 @@ In a private Exchange instance, asset administrators have access to change the v
...
+. Click *Add new instance*.
+. In the *Non-managed instances* section, enter the instance name, URL, and select a visibility option and click image:ex-checkmark-icon.png["checkmark icon",3%,3%].
```
Comment (hannanelson) [inline modules/ROOT/pages/to-configure-api-settings.adoc]:
```suggestion
. In the *Non-managed instances* section, enter the instance name, URL, select a visibility option, and click image:ex-checkmark-icon.png["checkmark icon",3%,3%].
```

File: modules/ROOT/pages/to-configure-api-settings.adoc [hunk 5]
```diff
@@ -16,6 +16,18 @@ In a private Exchange instance, asset administrators have access to change the v
 
 REST APIs provide a mocking service with data to test API features. The mocking service is always public.
 
+== Configure Non-Managed Instances 
+Exchange enables you to configure APIs with instances not managed by API Manager. Add, edit, and delete non-managed instances that are serving AsyncAPIs, GraphQL, REST, and HTTP APIs. You can change the name, URL, and visibility of a non-managed instances.
+
```
Comment (hannanelson) [inline modules/ROOT/pages/to-configure-api-settings.adoc:21]: Is this three separate tasks? Add, edit, and delete? If so, suggest separating them into different procedures so it doesn't imply you do all the steps.

You could use a bullet or definition list:

* Add an instance:
   a. blah
* Edit...
   a. blah

File: modules/ROOT/pages/to-configure-api-settings.adoc (hunk 3 above)
Comment (JennyHajee) [reply modules/ROOT/pages/to-configure-api-settings.adoc:25]: @hannanelson yes, left nav

File: modules/ROOT/pages/to-configure-api-settings.adoc (hunk 5 above)
Comment (JennyHajee) [reply modules/ROOT/pages/to-configure-api-settings.adoc:21]: good idea

File: modules/ROOT/pages/to-configure-api-settings.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/to-configure-api-settings.adoc:25]: OK, the style is to call that the "sidebar".

**Review by hannanelson — APPROVED** [review]

**Pull request merged**